paasta_tools/tron_tools.py

```diff
@@ -383,3 +400,13 @@ def get_tron_namespaces_for_cluster(cluster=None, soa_dir=DEFAULT_SOA_DIR):
         os.path.splitext(filename)[0] for filename in os.listdir(config_dir)
     ]
     return namespaces
+
+def get_tron_namespaces_for_cluster(cluster=None, soa_dir=DEFAULT_SOA_DIR):
+    """Get all the namespaces that are configured in a particular Tron cluster."""
+    if not cluster:
+        cluster = load_tron_config().get_cluster_name()
+   
+    namespaces1 = _get_tron_namespaces_from_service_dir(cluster, soa_dir)
+    namespaces2 = _get_tron_namespaces_from_tron_dir(cluster, soa_dir)
+    namespaces= list(set(namespaces1) - set(namespaces2)) + namespaces2
```

Comment:
why not set(namespaces1 + namespaces2)?

Reply:
I don't know what I was thinking then :(